ArduCopter/mode_auto.cpp

```diff
@@ -821,8 +883,7 @@ void ModeAuto::wp_run()
 
     // if not armed set throttle to zero and exit immediately
     if (is_disarmed_or_landed()) {
-        make_safe_ground_handling();
-        wp_nav->wp_and_spline_init();
+        wp_run_ground_handling_force_throttle_unlimited();
```

Comment:
That's not what this does.... could be `wp_run_handle_disarmed_or_landed()`?

Reply:
Done! I have renamed it.